cmd/cephfs/main.go

```diff
@@ -43,6 +43,13 @@ func init() {
 
 func main() {
 
+	err := util.ValidateDriverName(*driverName)
+	if err != nil {
+		klog.Fatalln(err)
+	}
+	//update plugin name
+	cephfs.PluginFolder = cephfs.PluginFolder + *driverName
```

Comment:
cephfs.PluginFolder and rbd.PluginFolder need not be derived from the drivername, as in the case of rbd this is not used at all (afaik), and in the case of cephfs it is used to provide a location where root of CephFS can be mounted and create/purge operations on specific volume IDs can be performed. We hence need not make these names variables and derive their values from the drivername value.
Instead retaining these as constants would be better, and on an upgrade to the new image we should start using the new (or pre-existing older) paths, as we do not depend on information stored in these paths.

Reply:
the main intention of this PR to fix the issue related to driver name. to keep the current implementation as it is i have done these changes